business/references/k8shttproute_references.go

```diff
@@ -74,13 +74,13 @@ func (n K8sHTTPRouteReferences) getConfigReferences(rt *k8s_networking_v1.HTTPRo
 	return result
 }
 
-func getAllK8sGateways(rt *k8s_networking_v1.HTTPRoute) []models.IstioReference {
+func getAllK8sGateways(prs []k8s_networking_v1.ParentReference, ns string) []models.IstioReference {
 	allGateways := make([]models.IstioReference, 0)
 
-	if len(rt.Spec.ParentRefs) > 0 {
-		for _, parentRef := range rt.Spec.ParentRefs {
+	if len(prs) > 0 {
```

Comment:
Same comment, I don't think you need this `if`.

Reply:
right